app/src/main/res/menu/activity_data_logger_menu.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:title="@string/import_log_action"
+        android:id="@+id/action_import_log"
+        app:showAsAction="never"
```

Comment:
Since there's only one menu item, let's keep it showing. Please remove this `never` part.